scripts/generate_tilesets.py

```diff
@@ -158,9 +130,16 @@ def check_aoi_tiles(aoi_tiles_gdf):
     
     ORTHO_WS_TYPE = cfg['datasets']['orthophotos_web_service']['type']
     ORTHO_WS_URL = cfg['datasets']['orthophotos_web_service']['url']
-    ORTHO_WS_SRS = cfg['datasets']['orthophotos_web_service']['srs']
+    if ORTHO_WS_TYPE != 'XYZ':
+        ORTHO_WS_SRS = cfg['datasets']['orthophotos_web_service']['srs']
+    else:
+        ORTHO_WS_SRS = "EPSG:3857" # <- NOTE: this is hard-coded
     if 'layers' in cfg['datasets']['orthophotos_web_service'].keys():
         ORTHO_WS_LAYERS = cfg['datasets']['orthophotos_web_service']['layers']
+    if 'parameters' in cfg['datasets']['orthophotos_web_service'].keys():
+        ORTHO_WS_PARAMETERS=cfg['datasets']['orthophotos_web_service']['parameters']
+    else:
+        ORTHO_WS_PARAMETERS={}
```

Comment:
Cette partie est un reliquat du fromattage que j'avais utilisé initialement pour le lien. Le plus simple serait de l'effacer, sauf si on veut reprendre ma proposition de passer les paramètres des requêtes XYZ et WMS en tant qu'un dictionnaire de paramètres.

Reply:
You're right!